### ISSUE TITLE
First type on Firefox

### ISSUE THREAD

**Issue by kalerosky** [Alex-D/Trumbowyg#496]
On Firefox when the box is empty and you're going to write for first time, every key you push is put in a new \<p\>.
In Chrome and Edge works fine.

### Informations

**Browser + version**: Firefox 51.0.1
**OS**: Windows 7/8/10 
**Resolution**: 1280x1024

### How to reproduce the bug?
Simply open this html with Firefox:

```html
<!DOCTYPE html\>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>    
    <link rel="Stylesheet" href="Styles/trumbowyg.min.css"/>
    <script type="text/javascript" src="Scripts/jquery-3.1.1.min.js"></script>
    <script type="text/javascript" src="Scripts/trumbowyg.min.js"></script>
    <script type="text/javascript">
        $(document).ready(function () { $("#ChipAhoi").trumbowyg({}); });
    </script>
</head>
<body>
    <form id="form1">
        <div>
            <textarea id="ChipAhoi"></textarea>
        </div>
    </form>
</body>
</html>
```

**Comment by martoriro** [comment]
@Alex-D 
Hi Alex, i know it's just been reported but do you have any update about when it could be fixed?
Thanks !

**Comment by martoriro** [comment]
Just discovered that disabling semantic tags seems to fix the issue

**Comment by Alex-D** [comment]
Yep, it's due to semantic wrap in p

**Comment by teamayu** [comment]
If you disable the semantics you get two sets of tags similar on the toolbar.

![20-02-2017-142420](https://cloud.githubusercontent.com/assets/10624092/23114182/78824440-f778-11e6-96bb-6fd7d068522b.png)

**Comment by Alex-D** [comment]
Juste need to change this from `4` to `3`
https://github.com/Alex-D/Trumbowyg/blob/develop/src/trumbowyg.js#L425

**Comment by teamayu** [comment]
Thank you Alex. Great plugin btw.

**Comment by istrau2** [comment]
+1, my users are complaining about this.

**Comment by yprez** [comment]
Looks like a duplicate of 502 which was fixed

**Comment by teamayu** [comment]
Semantics issue not fixed in 2.7.0

**Comment by Alex-D** [comment]
Seems to be a Firefox bug since I already call: 

```javascript
t.doc.execCommand('defaultParagraphSeparator', false, 'p');
```
